stable/datadog/values.yaml

```diff
@@ -499,3 +499,47 @@ deployment:
   ## Sets PriorityClassName if defined.
   #
   #  priorityClassName:
+
+clusterchecksDeployment:
+
+  ## @param enabled - boolean - required
+  ## If true, deploys agent dedicated for running the Cluster Checks instead of running in the Daemonset's agents.
+  ## ref: https://docs.datadoghq.com/agent/autodiscovery/clusterchecks/
+  #
+  enabled: false
+
+  ## @param replicas - integer - required
+  ## If you want to deploy the cluckerchecks agent in HA, keep at least clusterchecksDeployment.replicas set to 2.
+  ## And increase the clusterchecksDeployment.replicas according to the number of Cluster Checks.
+  #
+  replicas: 2
+
+  ## @param resources - object -required
+  ## Datadog clusterchecks-agent resource requests and limits.
+  #
+  resources:
+    requests:
+      cpu: 200m
+      memory: 500Mi
+    limits:
+      cpu: 200m
+      memory: 500Mi
+
+  ## @param affinity - object - optional
+  ## Allow the ClusterChecks Deployment to schedule using affinity rules.
+  ## By default, ClusterChecks Deployment Pods are forced to run on different Nodes.
```

Comment:
We might want to be more assertive on it being a requirement, not a "nice default"